src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/CspProviderExt.cs

```diff
@@ -7,13 +7,19 @@
 using System.Security.Cryptography.X509Certificates;
 using Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted.Setup;
 using Xunit;
+#if NET50_OR_LATER
+using System.Runtime.Versioning;
+#endif
 
 namespace Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted
 {
     /// <summary>
     /// Always Encrypted public CspProvider Manual tests.
     /// TODO: These tests are marked as Windows only for now but should be run for all platforms once the Master Key is accessible to this app from Azure Key Vault.
     /// </summary>
+#if NET50_OR_LATER
+    [SupportedOSPlatform("windows")]
```

Comment:
Won't the following `PlatformSpecific(TestPlatforms.Windows)` be sufficient?

Reply:
No. That's our own Attribute from XUnit extensions, .NET 5 doesn't recognize it.